requirements.txt

```diff
@@ -14,6 +14,7 @@ ccxt==1.21.6
 # CLI requirements
 click==7.0
 click-completion==0.5.2
+click-shell==1.0
```

Comment:
new requirement ?

Reply:
It was a missing requirement from `cli_app.py`.